pyproject.toml

```diff
@@ -33,6 +33,8 @@ pytest-mock = "^3.14.0"
 pytest-watcher = "^0.4.3"
 ruff = "^0.6.7"
 uvicorn = "^0.30.6"
+pydantic = "^2.9.2"
```

Comment:
Keep lists sorted

Reply:
I think poetry should manage this file, but this works:
`poetry self add poetry-sort && poetry sort`
So hopefully all my pyproject.toml files should be sorted from now on
(probably better put this in some tutorial but not sure which)